WBooks/Book Info/BookInfoViewController.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  DetailViewController.swift
+//  WBooks
+//
+//  Created by Gaston Maspero on 30/05/2019.
+//  Copyright © 2019 Wolox. All rights reserved.
+//
+
+import UIKit
+import WolmoCore
+
+class BookInfoViewController: UIViewController {
+    
+    private lazy var _view: BookInfoView = BookInfoView.loadFromNib()!
+    
+    private var _viewModel: BookInfoViewModel
+    
+    private var _commentsController: BookCommentsViewController
+    
+    private var _bookDetailsController: BookDetailsViewController
+    
+    init(viewModel: BookInfoViewModel, commentsController: BookCommentsViewController, bookDetailsController: BookDetailsViewController) {
+        _viewModel = viewModel
+        _commentsController = commentsController
+        _bookDetailsController = bookDetailsController
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    override var preferredStatusBarStyle: UIStatusBarStyle {
+        return .lightContent
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func loadView() {
+        view = _view
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setNeedsStatusBarAppearanceUpdate()
+        addNavBarButtons()
+        view.addSubview(_bookDetailsController.view)
+        view.addSubview(_commentsController.view)
+        setupLayout()
+    }
+    
+    private func setupLayout() {
+        //Book details
+        _bookDetailsController.view.translatesAutoresizingMaskIntoConstraints = false
+        _bookDetailsController.view.topAnchor.constraint(equalTo: topLayoutGuide.bottomAnchor, constant: 10).isActive = true
+        _bookDetailsController.view.leftAnchor.constraint(equalTo: view.leftAnchor, constant: 20).isActive = true
+        _bookDetailsController.view.rightAnchor.constraint(equalTo: view.rightAnchor, constant: -20).isActive = true
+        //Comments
+        _commentsController.view.topAnchor.constraint(equalTo: _bookDetailsController.view.bottomAnchor, constant: 10).isActive = true
+        _commentsController.view.leftAnchor.constraint(equalTo: view.leftAnchor, constant: 20).isActive = true
+        _commentsController.view.rightAnchor.constraint(equalTo: view.rightAnchor, constant: -20).isActive = true
+        _commentsController.view.bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: -20).isActive = true
```

Comment:
`20` seems to be a value that you are using constantly. It is better to set this kind of values in a constant property, so if it changes in the time you only need to change ir once.